External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -1306,8 +1306,13 @@ private static void AssignStatementToVar(ClassModel inClass, ScintillaControl sc
             template = TemplateUtils.ReplaceTemplateVariable(template, "Name", varname);
             template = TemplateUtils.ReplaceTemplateVariable(template, "Type", cleanType);
 
-            int indent = sci.GetLineIndentation(lineNum);
-            int pos = sci.PositionFromLine(lineNum) + indent / sci.Indent;
+            var indent = 0;
+            foreach (var c in line)
+            {
+                if (c == '\t' || c == ' ') indent++;
+                else break;
+            }
+            int pos = sci.PositionFromLine(lineNum) + indent;
```

Comment:
I still get the same result with your logic and with a simple sci.LineIndentPosition...